src/qt/res/css/style.css

```diff
@@ -436,6 +436,14 @@ QScrollBar:right-arrow {
     height: 10px;
 }
 
+/*******************************/
+/* SPLIT BLOCK CHECK BOX       */
+/*******************************/
+
+QDialog#SendCoinsDialog .QFrame#frameCoinControl .QCheckBox#splitBlockCheckBox {
+    color:#ffffff;
```

Comment:
this renders the checkbox label as white (on white), you can safely remove this block

Reply:
Removed